### ISSUE TITLE
Hi! TypeError.. 

### ISSUE THREAD

**Issue by 54853315** [suda/tool-bar-main#17]
I try install this package ,Atom show me this  :

`
Uncaught TypeError: this.element.querySelector is not a function
/Users/mac/.atom/packages/tool-bar-main/node_modules/atom-package-deps/lib/view.js:41
Hide Stack Trace
TypeError: this.element.querySelector is not a function
    at View.advance (/Users/mac/.atom/packages/tool-bar-main/node_modules/atom-package-deps/lib/view.js:41:36)
    at /Users/mac/.atom/packages/tool-bar-main/node_modules/atom-package-deps/lib/main.js:53:16
    at stdout (/Users/mac/.atom/packages/tool-bar-main/node_modules/atom-package-deps/lib/helpers.js:24:11)
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:212:1)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:543:20)
`

**Comment by deankevorkian** [comment]
I too have this issue, using Atom beta 1.22.0-beta1 on Windows 10. Is this going to be addressed?

**Comment by suda** [comment]
@54853315 & @deankevorkian when you update `tool-bar` and `toolbar-main` packages does this issue still persists?

**Comment by deankevorkian** [comment]
hi @suda! thanks for the reply. 
First of all - I forgot about this issue - it appears to have been fixed by the Atom core team, as it was a problem with Atom itself. I remember it stopped after I updated an Atom version, and I have a feeling it's related to decaffinating some package. Back at the time I thought it was related to [this Atom issue](https://github.com/atom/atom/issues/16135) but I didn't dive into the code to see if it's at all relevant.
Maybe @jasonrudolph could elaborate more on this.

However, updating both packages presents a new issue having to do with tool-bar.less. I'm using the latest Atom beta version (1.24.0-beta3). It's actually an issue with tool-bar itself and not tool-bar-main.

I'll paste the stacktrace here anyways.

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.24.0-beta3 x64
**Electron**: 1.6.15
**OS**: Microsoft Windows 10 Education
**Thrown From**: [tool-bar](ssh://git@github.com/suda/toolbar) package 1.1.5


### Stack Trace

Failed to load the tool-bar package

```
At variable @button-text-color-selected is undefined in C:\Users\Dean\.atom\packages\tool-bar\styles\tool-bar.less:37:13

LessError: variable @button-text-color-selected is undefined
at C:\Users\Dean\.atom\packages\tool-bar\styles\tool-bar.less:37:13
```

### Commands

```
```

### Non-Core Packages

```
apex-syntax 0.2.0 
apex-ui 1.0.3 
atom-ide-ui 0.8.1 
atom-material-syntax 1.0.8 
atom-material-ui 2.1.1 
highlight-selected 0.13.1 
ide-csharp 0.6.1 
ide-flowtype 0.18.0 
ide-typescript 0.7.2 
minimap 4.29.7 
monokai-seti 0.9.0 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
nucleus-dark-ui 0.12.3 
platformio-ide-terminal 2.8.0 
project-manager 3.3.5 
seti-icons 1.5.4 
seti-syntax 1.1.3 
solarized-minimal-dark-syntax 0.2.2 
solarized-minimal-light-syntax 0.2.2 
sync-settings 0.8.3 
tool-bar 1.1.5 
tool-bar-main 0.0.11 
```

If you'd like me to open the issue in the tool-bar repo I'd happily do so :)

And thanks for the reply!

**Comment by suda** [comment]
Great to know the original issue was fixed! The one you're mentioning was introduced in 1.1.5 but is now fixed in 1.1.6 :)
